content/en/user-manual/animation/anim-state-graph-assets.md

```diff
@@ -0,0 +1,171 @@
+
+These assets are therefore state machines for a model's animation behaviour and they control the flow of animation sequences over the lifecycle of an entity. A simple animstategraph asset used to define the behaviour of a wheel may define only two animation states; static and spinning. This asset can be defined to control when the wheel starts and stops spinning, for how long it will spin, the speed of the wheel spin and how sharply it starts / stops spinning. More advanced assets can be used to combine a multitude of animation states to create complex humanoid character animation behaviour.
+
+When selecting an animstategraph asset in the editors asset panel, you’ll open up the anim state graph editor view:
+
+![Initial Editor][1]
+
+Within this view you can edit your animation state graph. The following sections will highlight how different elements of the animstategraph asset can be used to define specific animation behaviour.
+
+## States
+
+In essence, states are used to specify which animations should play at a given point in time. An anim state graph can only be in one of these states at a given time.
+
+There are four types of states present in state graphs. Animation states, along with the START state, END state and ANY state. Only animation states can be created and deleted by the user and only these will be linked to animation assets. The other states are used to control the flow through the state machine.
+
+### Animation States
+
+![State][7]
+
+Animation states define a playable animation such as ‘Idle’, ‘Jump’ or ‘Walk’. New animation states can be created by right clicking on the blank canvas behind the state graph and selecting ‘Add new state’ from the menu. The editor will target your newly created state and show it’s inspector panel on the right hand side. Within this inspector the following state variables can be modified:
+
+
+<table class="table table-striped">
+    <tbody>
+        <tr>
+            <th>Variable</th>
+            <th>Description</th>
+        </tr>
+    <tr>
+        <td>Name</td>
+        <td>The name that this state should be called by. This is used to find and edit and play states via script. Names must be unique per state graph layer.</td>
+    </tr>
+    <tr>
+        <td>Speed</td>
+        <td>The playback speed for animations that are linked to this state.</td>
+    </tr>
+    <tr>
+        <td>Loop</td>
+        <td>Whether animations linked to this state should loop during playback. If set to false the animation will pause on it’s last keyframe until this state is exited.</td>
+    </tr>
+</tbody></table>
+
+### START state
+
+![Start State][8]
+
+The START state is the entrypoint of every state graph. When an anim component begins playing it’s assigned anim state graph, it will first enter this state and transition directly to the animation state it’s connected to. This animation state is called the default state and it can be selected via the layers panel here:
```

Comment:
```suggestion
The START state is the entry point of every state graph. When an anim component begins playing its assigned anim state graph, it will first enter this state and transition directly to the animation state it’s connected to. This animation state is called the default state and it can be selected via the layers panel here:
```